setconfig/RADIANT.json

```diff
@@ -1,13 +1,19 @@
 {
     "name": "RADIANT",
+
+    "default_args": {
+        "args": {
+            "use_uart": true
+        }
+    },
+
```

Comment:
Is that also necessary with your controller board?

Reply:
I don't think so. I will remove it, thanks!